03_letters/saurabhvyas9/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	frequencyLetters := make(map[rune]int)
+	for _, str := range s {
+		frequencyLetters[str]++
+	}
+	return frequencyLetters
+}
+
+func sortLetters(m map[rune]int) []string {
+	sortedLetters := make([]string, len(m))
+	tempKey := make([]int, len(m))
+	i := 0
+	for key := range m {
+		tempKey[i] = int(key)
+		i++
+	}
+	sort.Ints(tempKey)
```

Comment:
This is sorting on the rune value not the count. The lab was ambiguous in this regard so I'm fine with this solution. However I wonder, did you consider the other sort order? If so it would have been a good idea to document your assumption of the order to use, either as a comment in this function, or in the commit message.

No need to make any changes in response to this comment.

Reply:
Yes, the sorting is on the rune value. 
I did think of it when I was writing up the special test cases for emoticons etc.